code/modules/events/shuttle_catastrophe.dm

```diff
@@ -21,10 +21,20 @@
 	var/cause = pick("was attacked by [syndicate_name()] Operatives", "mysteriously teleported away", "had its refuelling crew mutiny",
 		"was found with its engines stolen", "\[REDACTED\]", "flew into the sunset, and melted", "learned something from a very wise cow, and left on its own",
 		"had cloning devices on it", "had its shuttle inspector put the shuttle in reverse instead of park, causing the shuttle to crash into the hangar")
+	var/message = "Your emergency shuttle [cause]. "
 
-	priority_announce("Your emergency shuttle [cause]. Your replacement shuttle will be the [new_shuttle.name] until further notice.", "CentCom Spacecraft Engineering")
+	if(SSshuttle.shuttle_insurance)
+		message += "Luckily, your shuttle insurance has covered the costs of repair!"
+		var/datum/bank_account/D = SSeconomy.get_dep_account(ACCOUNT_CAR)
+		if(D)
+			message += "You have been awarded a bonus from [command_name()] for smart spending."
+	else
+		message += "Your replacement shuttle will be the [new_shuttle.name] until further notice."
+	priority_announce(message, "CentCom Spacecraft Engineering")
```

Comment:
```suggestion
	priority_announce(message, "[command_name()] Spacecraft Engineering")
```
just in case someone renames centcom to "booger"

Reply:
Agreed!